scripts/processing-output/duplicate-diff.py

```diff
@@ -0,0 +1,50 @@
+import argparse
+import json
+
+# Measure the effect of accidental duplication
+# Should not be useful soon!
```

Comment:
So is this still useful?

Reply:
Yeah, it's useful until we release deduplication